docker/localsettings.py

```diff
@@ -149,7 +149,6 @@
 
 _ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
 JAR_SIGN = {
```

Comment:
Can this whole `JAR_SIGN` dict go?

Reply:
I am not sure about that one either. There is more javarosa stuff around.

Reply:
The only references I see to it are in `jadjar.py`, which is deleted in this PR.

Reply:
though I do think `commcare-cloud` does some stuff setting up JAR keys that could also be removed

Reply:
+1 to remove here. Most references in commcare-cloud are already [removed](https://github.com/dimagi/commcare-cloud/pull/4891) but here's a PR with a bit more cleanup: https://github.com/dimagi/commcare-cloud/pull/5948